tests/pytests/test_aggregate_count.py

```diff
@@ -0,0 +1,296 @@
+from common import *
+import bz2
+import json
+
+
+GAMES_JSON = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'games.json.bz2')
+
+def _setup_index_and_data(env, number_of_iterations=1):
+    env.cmd('FT.CREATE', 'games', 'ON', 'HASH',
+                        'SCHEMA', 'title', 'TEXT', 'SORTABLE',
+                        'brand', 'TEXT', 'NOSTEM', 'SORTABLE',
+                        'description', 'TEXT', 'price', 'NUMERIC',
+                        'categories', 'TAG')
+    con = env.getClusterConnectionIfNeeded()
+
+    # Use pipeline for much faster bulk HSET operations
+    pipeline = con.pipeline(transaction=False)
+    batch_size = 10000  # Execute pipeline every 10000 docs to avoid memory issues
+
+    j = 0
+    for i in range(number_of_iterations):
+        fp = bz2.BZ2File(GAMES_JSON, 'r')
+        for line in fp:
+            obj = json.loads(line)
+            id = obj['asin'] + (str(i) if i > 0 else '')
+            del obj['asin']
+            obj['price'] = obj.get('price') or 0
+            obj['categories'] = ','.join(obj['categories'])
+            cmd = ['HSET', id] + \
+                [str(x) if x is not None else '' for x in itertools.chain(
+                    *obj.items())]
+            pipeline.execute_command(*cmd)
+            j += 1
+            # Execute pipeline every batch_size docs to avoid memory issues
+            if j % batch_size == 0:
+                pipeline.execute()
+        fp.close()
+    pipeline.execute()
+
+
+def _get_total_results(res):
+    if isinstance(res, dict):
+        return res['total_results']
+    else:
+        return res[0]
+
+def _get_results(res):
+    if isinstance(res, dict):
+        return res['results']
+    else:
+        return res[1:]
+
+
+def _test_limit00(protocol):
+    env = Env(protocol=protocol)
+    add_values_iterations = 1
+    _setup_index_and_data(env, number_of_iterations=add_values_iterations)
+    docs = 2265 * add_values_iterations
+
+    queries_and_results = [
```

Comment:
Perhaps be explicit on the policy you're testing, so that in the future if the default changes the test is still good.